apps/webapp-source/Makefile

```diff
@@ -0,0 +1,32 @@
+PACKAGE = github.com/Azure-Samples/azure-sdk-for-go-samples/apps/basic_web_app
```

Comment:
Also did a quick check to see if `make` is called anywhere - it's not. This file is used for container/binary deploy only, not source, and it can be removed.